doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,172 @@
++=========================================================+========================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================+
+| ADDB                                                    | ADDB: Analysis and Diaganostics DataBase contains information describing ongoing activity of C2 system.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ATOM                                                    | Attribute on metadata server.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
So should I delete this?

Reply:
ATOM: /archaic/ "ATtribute On Meta-data server" is a design idea inherited from Lustre (q.v.). Some object attributes, such as the amount of space used by the object, can be naturally maintained on each service that keeps parts of the object with the "total" value of the attribute being obtained by querying all such services and aggregating results (e.g., by adding together space used by the object on each service). According to ATOM, such attributes should be also accumulated in some centralised meta-data location, to avoid the overhead of sending and processing queries to large number of services.